test/suites/basedir.bash

```diff
@@ -327,6 +327,34 @@ EOF
     expect_stat preprocessed_cache_hit 0
     expect_stat cache_miss 1
 
+    # -------------------------------------------------------------------------
+    TEST "-frecord-gcc-switches"
```

Comment:
Since this test (of the sloppiness option) doesn't really have anything to do with basedir, I suggest to add a few cases to the existing -frecord-gcc-switches test in base.bash instead.